CAToolsetPerfDataCollector/README.md

```diff
@@ -0,0 +1,96 @@
+---
+page_type: sample
+languages:
+- C++
+products:
+- CodeAnalysisPerfDataCollector
+description: ""
+urlFragment: "CAToolsetPerfDataCollector"
+---
+
+# Code Analysis Performance Data Collector
+
+![License](https://img.shields.io/badge/license-MIT-green.svg)
+
+This sample BuildInsights SDK application reads one or more Event Trace Log (ETW) files and prints summary of Code Analysis (CA) time relative to the overall execution time. It can also print execution time of each sub-process for each source file.  
+
+## Prerequisites
+
+This sample BuildInsights SDK application requires MSVC compiler and MSVC CA toolsets that are availabe in Visual Studio 2022 17.11 or later. It also requires BuildInsights SDK version 1.4.1 or later.
```

Comment:
**question**: The Visual Studio constraint is stricter than what is mentioned in the project-wide README (VS 2017 or later). Should we mention there that this project requires a bit more?

Reply:
Maybe we can have another column in the table with requirements? Or some kind of note?